packages/react-ai/src/components/AIConversation/AIConversation.tsx

```diff
@@ -72,6 +73,7 @@ function AIConversationBase({
       ...avatars,
     },
     isLoading,
+    allowAttachments,
```

Comment:
Do we need to add this on the `createAIConversation` as well?

Reply:
Was this comment resolved?